docs/contributing/provider_development_guide.md

```diff
@@ -194,7 +194,7 @@ type FavouriteDBInstanceParameters struct {
     // conventions trump cloud provider fidelity.
 
     // FancinessLevel specifies exactly how fancy this instance is.
-    FancinessLevel int `json:"fancinessLevel"`
+    FancinessLevel int `json:"fanciness_level"`
```

Comment:
@tomas-mota we use camel case in all of our fields, so I believe this is correct as-is

Reply:
maybe there is some implicit mapping, because the json it is referring to looks like this:
![image](https://user-images.githubusercontent.com/25929015/195091017-5cbac5e8-f3e7-44ef-9d08-69661897225e.png)

Reply:
I think the purpose is to demonstrate that we’ll translate snake case fields into camel case. See note:
```
// Note that fanciness_level becomes fancinessLevel below. Kubernetes API
     // conventions trump cloud provider fidelity.

```

Reply:
totally missed that comment, my bad!